lib/project_types/script/layers/infrastructure/script_project_repository.rb

```diff
@@ -115,6 +113,10 @@ def script_name
           project_config_value!("script_name")
         end
 
+        def description
+          project_config_value!("description")
```

Comment:
This should be the following because description is optional:

```suggestion
          project_config_value("description")
```